README.md

```diff
@@ -45,7 +45,7 @@
 
 ## Community
 
-- [Adafruit CircuitPython Discord channel #CircuitPython](https://discord.gg/adafruit) - 24/7 chat and support on CircuitPython including a weekly podcast.
+- [Adafruit CircuitPython Discord channel #CircuitPython](https://discord.com/invite/adafruit) - 24/7 chat and support on CircuitPython including a weekly podcast.
```

Comment:
The original link is preferred by the owner's.

Reply:
ok i will change that back soon

Reply:
done.

Reply:
The library repos currently link to `https://adafru.it/discord`. I'm not familiar enough with discord to know if there are differences between the different links though. It may have gone through a few different phases of link shortening type over the years.